### PR TITLE
Add IBM Cloud Private Istio multicluster deployment example.

### PR DESCRIPTION
**Pull request by gyliu513**
/cc @linsun @sdake @geeknoid

### CODE REVIEW

File: content/docs/examples/multicluster/icp/index.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
+---
+title: Enabling Multicluster on IBM Cloud Private
+description: Example multicluster IBM Cloud Private install of Istio.
+weight: 70
+keywords: [kubernetes,multicluster]
+---
+
+This example demonstrates how to use Istio' multicluster feature to join two
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: Istio's

File: content/docs/examples/multicluster/icp/index.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+[IBM Cloud Private](https://www.ibm.com/cloud/private) clusters together,
+using the [kubernetes multicluster installation instructions](/docs/setup/kubernetes/multicluster-install/).
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: capitalize Kubernetes

File: content/docs/examples/multicluster/icp/index.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+
+## Create the IBM Cloud Private Clusters
+
+1.  You can follow steps [here](https://www.ibm.com/support/knowledgecenter/en/SSBS6K_2.1.0.3/installing/installing.html)
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: `[Install two IBM Cloud Private servers](link)` and remove trailing text.

File: content/docs/examples/multicluster/icp/index.md [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+    to install two IBM Cloud Private clusters.
+    __NOTE__: Make sure individual cluster Pod CIDR ranges and service CIDR ranges must be unique across the multicluster
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: must be unique->are unique and do not overlap

File: content/docs/examples/multicluster/icp/index.md [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+    
+    {{< text bash >}}
+    ## Network in IPv4 CIDR format
+    network_cidr: 10.1.0.0/16
+
+    ## Kubernetes Settings
+    service_cluster_ip_range: 10.0.0.1/24
+    {{< /text >}}
+
+1.  After IBM Cloud Private cluster install finsihed, validate `kubectl` access to each cluster. Suppose we have
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: install finished->install finishes

File: content/docs/examples/multicluster/icp/index.md [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+    two clusters: cluster-1 and cluster-2.
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: Line break:

In this example, consider two clusters `cluster-1` and `cluster-2`:

File: content/docs/examples/multicluster/icp/index.md [hunk 7]
```diff
@@ -0,0 +1,219 @@
...
+
+    1.  Follow steps [here](https://www.ibm.com/support/knowledgecenter/SSBS6K_2.1.0.3/manage_cluster/cfc_cli.html)
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: `[`Configure cluster 1 with `kubectl`](link)`.

1. Check the cluster status:

File: content/docs/examples/multicluster/icp/index.md [hunk 8]
```diff
@@ -0,0 +1,219 @@
...
+        to config `kubectl` for cluster-1, and run following command to check cluster status.
+
+        {{< text bash >}}
+        $ kubectl get nodes
+        $ kubectl get pods --all-namespaces
+        {{< /text >}}
+
+    1.  Use same steps as cluster-1 above to check cluster-2.
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: 1. Use same steps as -> Repeat above steps to validate `cluster-2`.

File: content/docs/examples/multicluster/icp/index.md [hunk 9]
```diff
@@ -0,0 +1,219 @@
...
+    
+## Config Pod Communication Cross IBM Cloud Private Clusters
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: Cross/Across

File: content/docs/examples/multicluster/icp/index.md [hunk 10]
```diff
@@ -0,0 +1,219 @@
...
+
+IBM Cloud Private is using Calico Node-to-Node Mesh by default to manage container networks. The BGP Client
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: is using->uses

File: content/docs/examples/multicluster/icp/index.md [hunk 11]
```diff
@@ -0,0 +1,219 @@
...
+on each node distributes the IP router information to all nodes.
+
+In order to make sure pods can communicate across different clusters, you need to configure IP routers on
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: In order to make sure -> To ensure

File: content/docs/examples/multicluster/icp/index.md [hunk 12]
```diff
@@ -0,0 +1,219 @@
...
+all nodes in the cluster. You need to add IP routers in cluster-2 to cluster-1, and add IP routers in
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: Might break these into steps such as

1. Add IP routers from `cluster-1` to `cluster-2`
1. Add IP routers from `cluster-2` to `cluster-1`

File: content/docs/examples/multicluster/icp/index.md [hunk 13]
```diff
@@ -0,0 +1,219 @@
...
+cluster-1 to cluster-2.
+
+You can check how to add IP routers from cluster-1 to cluster-2. With Node-to-Node Mesh mode, each node will
+have IP routers connecting to peer nodes in the cluster. In this example, both cluster has three nodes.
+
+For cluster-1, the `hosts` file is as follows:
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: Full sentence can be shortened to The `hosts` file for cluster one:

File: content/docs/examples/multicluster/icp/index.md [hunk 14]
```diff
@@ -0,0 +1,219 @@
...
+
+{{< text bash >}}
+9.111.255.21 gyliu-icp-1
+9.111.255.129 gyliu-icp-2
+9.111.255.29 gyliu-icp-3
+{{< /text >}}
+
+For cluster-2, the `hosts` file is as follows:
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: See above comment.

File: content/docs/examples/multicluster/icp/index.md [hunk 15]
```diff
@@ -0,0 +1,219 @@
...
+
+{{< text bash >}}
+9.111.255.152 gyliu-ubuntu-3
+9.111.255.155 gyliu-ubuntu-2
+9.111.255.77 gyliu-ubuntu-1
+{{< /text >}}
+
+1.  Get all of the `ip route` on nodes in cluster-1 with command `ip route | grep bird`. See the following
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: Obtain routing information on all nodes in `cluster-` with the command `ip route | grep bird`.

Remove See the following example - its redundant.

File: content/docs/examples/multicluster/icp/index.md [hunk 16]
```diff
@@ -0,0 +1,219 @@
...
+    example. In this example, both cluster has three nodes.
+
+    {{< text bash >}}
+    root@gyliu-icp-1:~# ip route | grep bird
+    10.1.43.0/26 via 9.111.255.29 dev tunl0 proto bird onlink
+    10.1.158.192/26 via 9.111.255.129 dev tunl0 proto bird onlink
+    blackhole 10.1.198.128/26 proto bird
+    {{< /text >}}
+    {{< text bash >}}
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: I think you need a CR here.

File: content/docs/examples/multicluster/icp/index.md [hunk 17]
```diff
@@ -0,0 +1,219 @@
...
+    root@gyliu-icp-2:~# ip route | grep bird
+    10.1.43.0/26 via 9.111.255.29 dev tunl0  proto bird onlink
+    blackhole 10.1.158.192/26  proto bird
+    {{< /text >}}
+    {{< text bash >}}
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: another CR needed I suspect.

File: content/docs/examples/multicluster/icp/index.md [hunk 18]
```diff
@@ -0,0 +1,219 @@
...
+    10.1.198.128/26 via 9.111.255.21 dev tunl0  proto bird onlink
+    root@gyliu-icp-3:~# ip route | grep bird
+    blackhole 10.1.43.0/26  proto bird
+    10.1.158.192/26 via 9.111.255.129 dev tunl0  proto bird onlink
+    10.1.198.128/26 via 9.111.255.21 dev tunl0  proto bird onlink
+    {{< /text >}}
+
+1.  You can see there are three IP routers total for those three nodes in cluster-1.
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: Remove "you can see" and capitalze "There"

File: content/docs/examples/multicluster/icp/index.md [hunk 19]
```diff
@@ -0,0 +1,219 @@
...
+
+    {{< text bash >}}
+    10.1.158.192/26 via 9.111.255.129 dev tunl0  proto bird onlink
+    10.1.198.128/26 via 9.111.255.21 dev tunl0  proto bird onlink
+    10.1.43.0/26 via 9.111.255.29 dev tunl0  proto bird onlink
+    {{< /text >}}
+
+1.  Add those three IP routers to all nodes in cluster-2 by the command as follows.
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: change follows. to follows:

File: content/docs/examples/multicluster/icp/index.md [hunk 20]
```diff
@@ -0,0 +1,219 @@
...
+
+    {{< text bash >}}
+    $ ip route add 10.1.158.192/26 via 9.111.255.129 
+    $ ip route add 10.1.198.128/26 via 9.111.255.21
+    $ ip route add 10.1.43.0/26 via 9.111.255.29
+    {{< /text >}}
+
+1.  You can use same steps to add all IP routers from cluster-2 tocluster-1, as well. After the configuration
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: After the configuration finished, you will see -> After configuration is complete, all the pods

File: content/docs/examples/multicluster/icp/index.md [hunk 21]
```diff
@@ -0,0 +1,219 @@
...
+    finished, you will see that the pods in those two different clusters can communication with each other.
+
+1.  You can verify it by pinging pod IP in cluster-2 from cluster-1. The following is a pod from cluster-2 with
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: You can verify it -> Verify cross pod communication

File: content/docs/examples/multicluster/icp/index.md [hunk 22]
```diff
@@ -0,0 +1,219 @@
...
+    pod IP as 20.1.47.150.
+
+    {{< text bash >}}
+    root@gyliu-ubuntu-1:~/cluster# kubectl get pods -owide  -n kube-system | grep platform-ui
+    platform-ui-lqccp                                             1/1       Running     0          3d        20.1.47.150     9.111.255.77
+    {{< /text >}}
+
+1.  In one of cluster-1 node, ping this pod IP, it should succeed.
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: From a node in `cluster-1` ping the pod IP which should succeed..

File: content/docs/examples/multicluster/icp/index.md [hunk 23]
```diff
@@ -0,0 +1,219 @@
...
+
+    {{< text bash >}}
+    [root@gyliu-icp-1 mc]# ping 20.1.47.150
+    PING 20.1.47.150 (20.1.47.150) 56(84) bytes of data.
+    64 bytes from 20.1.47.150: icmp_seq=1 ttl=63 time=0.759 ms
+    {{< /text >}}
+
+The above configuration actually configured full IP route mesh across all nodes in the two IBM Cloud Private Clusters,
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: actually->completely

File: content/docs/examples/multicluster/icp/index.md [hunk 24]
```diff
@@ -0,0 +1,219 @@
...
+which enabled Pod communication cross clusters.
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: enabled/enables

File: content/docs/examples/multicluster/icp/index.md [hunk 25]
```diff
@@ -0,0 +1,219 @@
...
+
+## Install the Istio for multi cluster
+
+You can follow steps [here](/docs/setup/kubernetes/multicluster-install/) to install and config local Istio control plane
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: recommend avoiding `[here]` and instead use something more descriptive such as `Follow the multicluster installation steps(link)`

File: content/docs/examples/multicluster/icp/index.md [hunk 26]
```diff
@@ -0,0 +1,219 @@
...
+and Istio remote on cluster-1 and cluster-2.
+
+In this example, I was using cluster-1 as local Istio control plane and cluster-2 as Istio remote.
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: In this example, I was using -> This example uses cluster-1

File: content/docs/examples/multicluster/icp/index.md [hunk 27]
```diff
@@ -0,0 +1,219 @@
...
+
+## Deploy Bookinfo Example Across Clusters
+
+__NOTE__: The following example is enabling [sidecar automatic injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection).
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: is enabling->enables.

File: content/docs/examples/multicluster/icp/index.md [hunk 28]
```diff
@@ -0,0 +1,219 @@
...
+
+1.  Install bookinfo on the cluster-1. Remove the `reviews-v3` deployment to deploy on remote:
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: all instances of cluster-1 and cluster-2 should have backticks

File: content/docs/examples/multicluster/icp/index.md [hunk 29]
```diff
@@ -0,0 +1,219 @@
...
+    {{< text bash >}}
+    $ kube apply -f $HOME/reviews-v3.yaml
+    {{< /text >}}
+
+1.  Get the `istio-ingressgateway`'s `INGRESS_HOST` and `INGRESS_PORT` variables for accessing the gateway via
+    [these instructions](/docs/tasks/traffic-management/ingress/#determining-the-ingress-ip-and-ports).
+
+    Access `http://<INGRESS_HOST>:<INGRESS_PORT>/productpage` repeatedly and each version of reviews should be equally
+    loadbalanced, including `reviews-v3` in the remote cluster (red stars). It may take several accesses (dozens) to
+    demonstrate the equal loadbalancing between `reviews` versions.
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: demonstrate the equal -> demonstrate equal

File: content/docs/examples/multicluster/icp/index.md (hunk 1 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: done

File: content/docs/examples/multicluster/icp/index.md (hunk 2 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 3 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 4 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 5 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 6 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 8 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 9 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 10 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 11 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 12 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 13 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 14 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 15 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 16 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: I think you mean a new line here? ;-)

File: content/docs/examples/multicluster/icp/index.md (hunk 18 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 19 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 20 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 21 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 22 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 23 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 24 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 25 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 26 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 27 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 28 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 29 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md [hunk 30]
```diff
@@ -0,0 +1,225 @@
...
+    
+    {{< text bash >}}
+    ## Network in IPv4 CIDR format
+    network_cidr: 10.1.0.0/16
+
+    ## Kubernetes Settings
+    service_cluster_ip_range: 10.0.0.1/24
+    {{< /text >}}
+
+1.  After IBM Cloud Private cluster install finsihes, validate `kubectl` access to each cluster. In this example, consider
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: spelling - finishes

File: content/docs/examples/multicluster/icp/index.md [hunk 31]
```diff
@@ -0,0 +1,225 @@
...
+    [Configure cluster-1 with kubectl](//www.ibm.com/support/knowledgecenter/SSBS6K_2.1.0.3/manage_cluster/cfc_cli.html).
+
+    1.  Check the cluster status:
+
+        {{< text bash >}}
+        $ kubectl get nodes
+        $ kubectl get pods --all-namespaces
+        {{< /text >}}
+
+    1.  Repeat above steps to validate `cluster-2`.
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: something unsatisfying about this "Repeat above".  Could reference the setup of the IBM cloud private steps.  Might say "Repeat above two steps" or perhaps @rcaballeromx has a better way to represent this.

File: content/docs/examples/multicluster/icp/index.md [hunk 32]
```diff
@@ -0,0 +1,225 @@
...
+
+## Config Pod Communication Across IBM Cloud Private Clusters
+
+IBM Cloud Private uses Calico Node-to-Node Mesh by default to manage container networks. The BGP Client
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: BGP client might be more appropriate (lowercase client)

File: content/docs/examples/multicluster/icp/index.md [hunk 33]
```diff
@@ -0,0 +1,225 @@
...
+on each node distributes the IP router information to all nodes.
+
+To ensure pods can communicate across different clusters, you need to configure IP routers on all nodes
+in the cluster. You need two steps:
+
+1.  Add IP routers from `cluster-1` to `cluster-2`.
+
+1.  Add IP routers from `cluster-2` to `cluster-1`.
+
+You can check how to add IP routers from `cluster-1` to `cluster-2`. With Node-to-Node Mesh mode, each
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: "You can check how to add IP routers from c1 to c2" doesn't make sense in this paragraph.  I'm not sure how to reword it in composition with the second sentence.  Perhaps switch the order of the sentences and change "You can check how" to " Validate ..."

File: content/docs/examples/multicluster/icp/index.md [hunk 34]
```diff
@@ -0,0 +1,225 @@
...
+
+1.  Add those three IP routers to all nodes in `cluster-2` by the command to follows:
+
+    {{< text bash >}}
+    $ ip route add 10.1.158.192/26 via 9.111.255.129 
+    $ ip route add 10.1.198.128/26 via 9.111.255.21
+    $ ip route add 10.1.43.0/26 via 9.111.255.29
+    {{< /text >}}
+
+1.  You can use same steps to add all IP routers from `cluster-2` to `cluster-1`, as well. After configuration
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: use same steps -> use the same steps.  Remove , as well.

File: content/docs/examples/multicluster/icp/index.md [hunk 35]
```diff
@@ -0,0 +1,225 @@
...
+
+1.  From a node in `cluster-1` ping the pod IP which should succeed.
+
+    {{< text bash >}}
+    [root@gyliu-icp-1 mc]# ping 20.1.47.150
+    PING 20.1.47.150 (20.1.47.150) 56(84) bytes of data.
+    64 bytes from 20.1.47.150: icmp_seq=1 ttl=63 time=0.759 ms
+    {{< /text >}}
+
+The above configuration completely configured full IP route mesh across all nodes in the two IBM Cloud Private Clusters,
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: The steps above (again, which above are we referencing?) enables Pod communication across clusters by configuring a full IP routing mesh across all nodes in the two IBM Cloud Private Clusters.

File: content/docs/examples/multicluster/icp/index.md [hunk 36]
```diff
@@ -0,0 +1,225 @@
...
+which enables Pod communication cross clusters.
+
+## Install the Istio for multi cluster
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: the Istio->Istio

File: content/docs/examples/multicluster/icp/index.md [hunk 37]
```diff
@@ -0,0 +1,225 @@
...
+
+[Follow the multicluster installation steps](/docs/setup/kubernetes/multicluster-install/) to install and config
+local Istio control plane and Istio remote on `cluster-1` and `cluster-2`.
+
+This example uses `cluster-1` as local Istio control plane and `cluster-2` as Istio remote.
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: as local-> as the local  as Istio remove -> as the Istio remote.

File: content/docs/examples/multicluster/icp/index.md [hunk 38]
```diff
@@ -0,0 +1,225 @@
...
+
+## Deploy Bookinfo Example Across Clusters
+
+__NOTE__: The following example enables [sidecar automatic injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection).
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: change sidecar automatic injection to "automatic sidecar injection"

File: content/docs/examples/multicluster/icp/index.md [hunk 39]
```diff
@@ -0,0 +1,225 @@
...
+
+1.  Install bookinfo on the `cluster-1`. Remove the `reviews-v3` deployment to deploy on remote:
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: on the -> on `cluster-1`

Comment (sdake) [reply content/docs/examples/multicluster/icp/index.md]: Remove the `reviews-v3` -> Remove `reviews-v3`

File: content/docs/examples/multicluster/icp/index.md [hunk 40]
```diff
@@ -0,0 +1,225 @@
...
+            image: istio/examples-bookinfo-reviews-v3:1.5.0
+            imagePullPolicy: IfNotPresent
+            ports:
+            - containerPort: 9080
+    EOF
+    {{< /text >}}
+
+    _Note:_ The `ratings` service definition is added to the remote cluster because `reviews-v3` is a
+    client of `ratings` and creating the service object creates a DNS entry.  The Istio sidecar in the
+    `reviews-v3` pod will determine the proper `ratings` endpoint after the DNS lookup is resolved to a
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: I think we need a "how multicluster works blog" :)

File: content/docs/examples/multicluster/icp/index.md [hunk 41]
```diff
@@ -0,0 +1,225 @@
...
+    service address.  This would not be necessary if a multicluster DNS solution were additionally set up, e.g. as
+    in a federated Kubernetes environment.
+
+1.  Install the `reviews-v3` deployment on the remote `cluster-2`.
+
+    {{< text bash >}}
+    $ kube apply -f $HOME/reviews-v3.yaml
+    {{< /text >}}
+
+1.  Get the `istio-ingressgateway`'s `INGRESS_HOST` and `INGRESS_PORT` variables for accessing the gateway via
```
Comment (sdake) [inline content/docs/examples/multicluster/icp/index.md]: `[Determine  the ingress IP and ports](link) for `istio-ingressgateway`s ...   end goal is to get rid of "these instructions" and replace it  with something more descriptive.

**Comment by sdake** [comment]
please get gate in a green state - 👍

File: content/docs/examples/multicluster/icp/index.md (hunk 30 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 33 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done, updated to "You can check how to add IP routers from `cluster-1` to `cluster-2` to validate pod to pod communication across clusters."

File: content/docs/examples/multicluster/icp/index.md (hunk 34 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 32 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 35 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Updated to 

"The steps in this section enables Pod communication across clusters by configuring a full IP routing mesh across all nodes in the two IBM Cloud Private Clusters."

File: content/docs/examples/multicluster/icp/index.md (hunk 36 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 37 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 38 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 39 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

File: content/docs/examples/multicluster/icp/index.md (hunk 40 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: @sdake do you mean this https://github.com/istio/istio.github.io/issues/1381 ? Add a detailed use case for multicluster?

File: content/docs/examples/multicluster/icp/index.md (hunk 41 above)
Comment (gyliu513) [reply content/docs/examples/multicluster/icp/index.md]: Done

**Comment by gyliu513** [comment]
@sdake @geeknoid @linsun this should be land in 1.0 release.

File: content/docs/examples/multicluster/icp/index.md (hunk 40 above)
Comment (sdake) [reply content/docs/examples/multicluster/icp/index.md]: Yup, that is a separate task - thanks i forgot that was in the issue queue.

**Comment by gyliu513** [comment]
Thanks @sdake 👍

**Pull request merged**